app/views/layouts/application.html.haml

```diff
@@ -51,7 +51,7 @@
               = fa_icon "cog fw"
               Settings
         %hr
-    #content.container
+    %div#content.container{ :style => "width:auto; height:100%" }
```

Comment:
`%div` is the default tag, so it's better to leave it off. Again, put this inline style in an actual CSS class.